base/ca/src/main/java/org/dogtagpki/server/ca/cli/CACertImportCLI.java

```diff
@@ -112,8 +112,6 @@ public void execute(CommandLine cmd) throws Exception {
             throw new Exception("Unsupported format: " + certFormat);
         }
```

Comment:
Yes, I've moved the `byte[]`-related code down, but git thought that I moved JSS initialization up, so the patch generated by git doesn't really reflect the changes I'm making, but that should be fine.

Since we need to cherry-pick these changes to `v11.2` branch I'd rather keep them simple, but we can do more cleanups in the `master` branch later.